memstore/list/live_vector_party.go

```diff
@@ -71,9 +89,6 @@ func (vp *LiveVectorParty) SafeDestruct() {
 			vp.memoryPool.Destroy()
 			vp.memoryPool = nil
 		}
-		if totalBytes > 0 {
-			vp.reporter(-totalBytes)
```

Comment:
where do we move this reporting to?

Reply:
it will be treated the same as all other types of VP, report from livestore.purgeBatch